README.md

```diff
@@ -150,7 +1444,12 @@ vars:
   hubspot_service_enabled: true                           # Enables all service/ticket models. Default = false
   hubspot_ticket_deal_enabled: true                       # Default = false
 ```
+
+### Dbt-core Version Requirement for disabling freshness tests
+If you are not using a source table that involves freshness tests, please be aware that the feature to disable freshness was only introduced in dbt-core 1.1.0. Therefore ensure the dbt version you're using is v1.1.0 or greater for this config to work.
```

Comment:
Actually, I just realized this package requires the use of dbt v1.3.0 or higher. So I think we may be able to remove this messaging in both the source and transform.

Reply:
removed